.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Typing Dev Container",
+    "build": {
+        "dockerfile": "Dockerfile"
+    },
+    "settings": { 
+        "terminal.integrated.shell.linux": "/bin/zsh",
+        "go.gopath": "/usr/local/go"
```

Comment:
given that GOROOT is /usr/local/go (according to Dockerfile), I think this setting is strange.
go.gopath will. be unnecessary (or set to $HOME/go or something)